test/stepDefinitions/searchResultSteps.js

```diff
@@ -0,0 +1,120 @@
+
+let SearchResultPage = require('../pageObjects/searchResultPage.js');
+let TestData = require('../utils/TestData.js');
+let chai = require("chai").use(require("chai-as-promised"));
+let baseSteps = require('./baseSteps.js');
+let expect = chai.expect;
+let Data = require('../utils/TestData.js');
+
+var { defineSupportCode } = require("cucumber");
+
+defineSupportCode(function ({ Given, When, Then, Before, After }) {
+
+  let searchResultPage = new SearchResultPage();
+  const valueArr = ['Odin', 'Freya', 'Demeter', 'Quetzcuatl', 'Thor', 'Zeus', 'Wotan', 'Ares', 'Zephyr', 'Baldur', 'Aegir', 'Heimdall', 'Loki', 'Tyr', 'Hera', 'Dionysus', 'Poseidon', 'Mithras', 'Khantengri', 'Ishtar', 'Ahuramazda', 'Aphrodite', 'Crom', 'Lady of the Lake', 'Morgana', 'Baphomet'];
+
+  async function bulkUpdateCases(arr) {
+    let columnResults = await searchResultPage.getCaseListComponent().getFirstColumnResults();
+    Data.jurisdiction = 'Auto Test 1';
```

Comment:
I think we should try and set this data as more of a pre requisite of the test rather than in the middle of the test. we can add another bdd step, can reuse one of the ones from definitionFileSteps.js and put it in the Backround section of the bdd then the data will be pre set

Reply:
`Given a case type containing every field type exists` would work